lib/octokit/default.rb

```diff
@@ -143,7 +143,7 @@ def ssl_verify_mode
         # 0 is OpenSSL::SSL::VERIFY_NONE
         # 1 is OpenSSL::SSL::SSL_VERIFY_PEER
         # the standard default for SSL is SSL_VERIFY_PEER which requires a server certificate check on the client
-        ENV['OCTOKIT_SSL_VERIFY_MODE'] || 1 
+        ENV.has_key?('OCTOKIT_SSL_VERIFY_MODE')? ENV['OCTOKIT_SSL_VERIFY_MODE'].to_i : 1
```

Comment:
Instead of checking if key is present, I'd rather see something like:

```ruby
ENV.fetch('OCTOKIT_SSL_VERIFY_MODE', 1).to_i
```

Reply:
I had already rebased with a replacement. Do you want your line exactly?

Reply:
If you wouldn't mind yeah I kind of prefer using `ENV#fetch`

Reply:
done